src/Explorer.js

```diff
@@ -879,19 +903,63 @@ function defaultArgs(
 
 class FieldView extends React.PureComponent<FieldViewProps, {}> {
   _previousSelection: ?SelectionNode;
-  _addFieldToSelections = () =>
-    this.props.modifySelections([
-      ...this.props.selections,
-      this._previousSelection || {
-        kind: 'Field',
-        name: {kind: 'Name', value: this.props.field.name},
-        arguments: defaultArgs(
-          this.props.getDefaultScalarArgValue,
-          this.props.makeDefaultArg,
-          this.props.field,
-        ),
-      },
-    ]);
+  _addFieldToSelections = event => {
+    const fieldType = getNamedType(this.props.field.type);
+    const rawSubfields = isObjectType(fieldType) && fieldType.getFields();
+
+    const shouldSelectAllSubfields = !!rawSubfields && event.altKey;
+
+    const subFields: Array<FieldNode> = !!rawSubfields
+      ? Object.keys(rawSubfields).map(fieldName => {
+          return {
+            kind: 'Field',
+            name: {kind: 'Name', value: fieldName},
+            arguments: [],
+          };
+        })
+      : [];
+
+    const subSelectionSet: SelectionSetNode = {
+      kind: 'SelectionSet',
+      selections: subFields || [],
```

Comment:
Do you need the `||`?

Reply:
Should be covered by the definition of `subFields`, was probably overly-defensive